scripts/dracula.sh

```diff
@@ -135,114 +116,86 @@ main()
   # wait unit $datafile exists just to avoid errors
   # this should almost never need to wait unless something unexpected occurs
   while $show_weather && [ ! -f $datafile ]; do
```

Comment:
See other comment.

Reply:
I moved this logic in the plugin weather condition